docs/release-notes/NuGet-6.12.md

```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.11.1
+
+NuGet 6.12.1 is available in Visual Studio 17.12.0 and the.NET 9.0.101 SDK.
+
+### Issues fixed in this release
+
+* Deserializing an empty version range in a package dependency fails in .NET SDK 9.0.100-rc.2 [#13869](https://github.com/NuGet/Home/issues/13869)
+
+## Summary: What's New in 6.12
```

Comment:
6.12 has an unlikely bug in the new resolver, https://github.com/NuGet/Home/issues/13890. 

Should we include it in these docs?
How should we include it there?

Reply:
I added a new section for known issues